_pages/rust-hack.md

```diff
@@ -50,17 +50,25 @@ Rust is a systems programming language that runs blazingly fast, prevents segfau
 
-1. Put an event team together. Going alone is fine, but at least two people is advisable. Get in touch with the [[Rust Community Team](mailto:community@rust-lang.org)] and check the [Rust Community Calendar](https://calendar.google.com/calendar/embed?src=apd9vmbc22egenmtu5l6c5jbfc@group.calendar.google.com) - maybe there's already people in your area.
+1. Put an event team together. Going alone is fine, but at least two people is advisable. Check the [Rust Community Calendar](https://calendar.google.com/calendar/embed?src=apd9vmbc22egenmtu5l6c5jbfc@group.calendar.google.com) - maybe there's already people in your area. You can also get in touch with  the [Rust Community Team](mailto:community@rust-lang.org) for additional support.
 2. Try to find people with Rust experience who want to help. If you find none, this is fine - many Hack and Learns started out by curious people.
-3. Choose a date and secure a venue. Fill the [event template](https://github.com/rust-community/events-team/issues/new?template=new_meetup.md&labels=meetup) for access to the calendar so that you can enter it there.  It will then be sent around with Rusts weekly newsletter and promoted in other fashions.
-4. Setup the event on Reps Portal ([instructions](https://wiki.mozilla.org/ReMo/SOPs/Event_hosting)). Please make sure the initiative is set to "MozActivate" and the functional area indicates "Rust".
-5. Get [Material](https://github.com/rust-community/resources) and print them to hand them out. You can also ask the [Rust Community Team](mailto:community@rust-lang.org) if they can send some stickers.
 
-For duration, we recommend at least 2-3 hours event with a minimum of 10 attendees.
 
-Some Mozillians have already done such sessions. For example our [Community Spaces](https://wiki.mozilla.org/Participation/Community_Spaces) have, so you can reach out the them to ask for some guidance.
+Some Mozillians have already done such sessions. For example see [past Mozilla Reps events](https://reps.mozilla.org/events/#/period/past/category/rust/).
+
+**2-4 Weeks Before the Event**
+
+1. Read the [event guide](/eventguide/) on how to set up an event page and how to organize an event.
+1. Choose a date and secure a venue.
+1. Fill the [event template](https://github.com/rust-community/events-team/issues/new?template=new_meetup.md&labels=meetup) for access to the calendar so that you can enter it there.  It will then be sent around with Rust's weekly newsletter and promoted in other fashions.
+1. Setup the event on the Reps Portal ([instructions](https://wiki.mozilla.org/ReMo/SOPs/Event_hosting)). Please make sure the initiative is set to "MozActivate" and the functional area indicates "Rust".
+
+**1 Week Before the Event**
+
+1. Get [Material](https://github.com/rust-community/resources) and print them to hand them out. You can also ask the [Rust Community Team](mailto:community@rust-lang.org) if they can send some stickers.
```

Comment:
We probably can't send stickers within a week. They should order earlier.

Reply:
Right, putting that earlier then